src/Tizen.Multimedia.MediaPlayer/Player/AudioPitch.cs

```diff
@@ -0,0 +1,125 @@
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Diagnostics;
+using static Interop;
+
+namespace Tizen.Multimedia
+{
+    /// <summary>
+    /// Provides the ability to control a audio pitch for <see cref="Multimedia.Player"/>.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class AudioPitch
+    {
+        /// <summary>
+        /// Gets the <see cref="Multimedia.Player"/> that owns this instance.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public Player Player { get; }
+
+        /// <summary>
+        /// Provides the means to retrieve audio pitch information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        internal AudioPitch(Player player)
+        {
+            Debug.Assert(player != null);
+            Player = player;
+        }
+
+        /// <summary>
+        /// Gets or sets the audio pitch.
+        /// </summary>
+        /// <remarks>This function is used for audio content only. The default value is false.</remarks>
+        /// <remarks>To set, the player must be in the <see cref="PlayerState.Idle"/> state.</remarks>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsEnabled
+        {
+            get
+            {
+                Player.ValidateNotDisposed();
+                NativePlayer.IsAudioPitchEnabled(Player.Handle, out var value).
+                    ThrowIfFailed(Player, "Failed to get whether the audio pitch is enabled or not");
+                return value;
+            }
+
+            set
+            {
+                Player.ValidateNotDisposed();
+                Player.ValidatePlayerState(PlayerState.Idle);
+
+                NativePlayer.SetAudioPitchEnabled(Player.Handle, value).
+                    ThrowIfFailed(Player, "Failed to enable the audio pitch of the player");
+            }
+        }
+
+        /// <summary>
+        /// Gets the audio pitch value.
+        /// </summary>
+        /// <returns>The pitch value of an audio stream. The default value is 1.</returns>
+        /// <remarks>This function is used for audio content only.</remarks>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <seealso cref="IsEnabled"/>
+        /// <seealso cref="SetValue(float)"/>
+        /// <since_tizen> 6 </since_tizen>
+        public float GetValue()
```

Comment:
Why do you use `Value` naming?
Why dou you use method instead of property?

How about use property namded by Pitch
```suggestion
        public float Pitch { get { ... } set {...} }
```

Reply:
ok i'll change it to property.
but i think naming still looks weird. because pitch will be written twice.

Player.AudioPitch.Pitch = 1.1;

Reply:
Ok, `Value` naming is reasonable for it